db/migrate_cards/20170124120401_research_page.rb

```diff
@@ -0,0 +1,31 @@
+# -*- encoding : utf-8 -*-
+
+class ResearchPage < Card::Migration
+  def up
+    ensure_card "Research Page", type_id: Card::SessionID,
+                codename: "research_page"
+
+    ensure_field_rules "Company"
+    ensure_field_rules "Metric"
+
+    ensure_card "Unknown", codename: "unknown"
+    ensure_card "Unknown+*right+*input", content: "checkbox"
+    ensure_card "Unknown+*right+*add help",
```

Comment:
Perhaps this should be `*help` and we should add this field to the edit view (in case corrections are needed)